README.md

```diff
@@ -3159,6 +3159,11 @@ In order to achieve greater coverage and encourage more people to contribute to
                     <img align="center" height="25" src="./logos/java.svg" />
                 </a>
             </td>
+            <td> <!-- kotlin -->
+                <a href="./src/kotlin/MergeSort.kt">
+                    <img align="center" height="25" src="./logos/kotlin.svg" />
+                </a>
+            </td>
```

Comment:
We already have a table data cell for Kotlin, please remove this one and update the correct reference, lines 3198 and 3199.